search/create_datastore_and_search.ipynb

```diff
@@ -0,0 +1,3120 @@
+{
```

Comment:
<p>I don't think you need to use Vertex AI LLMs for this. You can just do a basic search query or point to other notebooks/code samples that show how to do this.</p><p> </p><p>For example:</p><p> </p><p><a href="https://cloud.google.com/generative-ai-app-builder/docs/preview-search-results" target="_blank">https://cloud.google.com/generative-ai-app-builder/docs/preview-search-results</a></p><p> </p><p>And for retrieval augmented Generation with LangChain:</p><p> </p><p><a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/search/retrieval-augmented-generation/examples/question_answering.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/search/retrieval-augmented-generation/examples/question_answering.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>